src/content/pages/wasm.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: WebAssembly Summit
+subtitle: A summit to bring together maintainers and users of the Python with WebAssembly, to discuss the state of this ecosystem, existing challenges and ongoing work.
+---
+
+# WebAssembly Summit
+
+* **When**: Tuesday, July 9th.
+* **Where**: Prague Congress Centre (PCC), Room TBC
+* **Who can join**: Anyone with a valid in-person EuroPython 2024 ticket - Conference, Tutorial or Combined (see [ticket types](https://ep2024.europython.eu/tickets#ticket-types) for details).
+
+This summit aims to bring together maintainers and users of the Python with WebAssembly, to discuss the state of this ecosystem, existing challenges and ongoing work.
+
+Read about last year's summit [here](https://github.com/psf/webassembly/blob/main/meetings/2023/webassembly-summit-europython.md).
+
+
+## Agenda
+
+* 9:00: Meet and greet (many of us won’t know each other) + unconference-y post-it based organisation.
+* 9:30: Presentations (30mins each)
+* 11:00: Coffee
+* 11:20: Presentations
```

Comment:
```suggestion
* 11:15: Presentations
```
Feel free to keep your own schedule. The official coffee break is anounced to be from 11:00 to 11:15 .

Reply:
I want to ensure we're in sync. :+1: